src/main/java/gmail/mishchuk/modultest_2/PhoneBook.java

```diff
@@ -0,0 +1,64 @@
+package gmail.mishchuk.modultest_2;
+
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class PhoneBook {
+
+    private Map<String, String> contacts = new HashMap<>();
+
+    //можна було б і не створювати пустий конструктор
+    public PhoneBook() {
+    }
+
+    public void addContact(String name, String phone) {
+        contacts.put(name, phone);
+        System.out.println("Створено контакт:\n\t"
+                + name + ", номер: "
+                + phone);
+    }
+
+    public void updateContact(String name, String newPhone) {
+        if (contacts.containsKey(name)) {
+            contacts.put(name, newPhone);
+            System.out.println("\nКонтакт оновлено:\n\t"
+                    + name + ", новий номер: "
+                    + newPhone);
+        } else {
+            addContact(name, newPhone);
+        }
+    }
```

Comment:
Все ж таки краще, щоб `updateContact` не додавав контакт якщо його вже немає. Бо може виникнути ситуація коли контакта немає в Ви його оновлюєте. В такому випадку Ваш код додасть контакт знову що не дуже добре.

```suggestion
    public void updateContact(String name, String newPhone) {
        if (contacts.containsKey(name)) {
            contacts.put(name, newPhone);
            System.out.println("\nКонтакт оновлено:\n\t"
                    + name + ", новий номер: "
                    + newPhone);
        }
    }
```